compose/docker-compose.yml

```diff
@@ -153,19 +166,72 @@ services:
       - ../src/python:/home/broker/python
     working_dir: "/home/broker/python"
     entrypoint: [ "python", "-m", "delivery.UbidotsWriter" ]
+    ports:
+      - "8006:8006" # Expose the metrics port
 
   frred:
     image: broker/python-base
     restart: unless-stopped
     env_file:
       - .env
+    environment:
+      - PYTHONPATH=/home/broker/broker/src/python:/home/broker/broker/test/python
     depends_on:
-      db:
-        condition: "service_healthy"
-      mq:
-        condition: "service_healthy"
+      - "db"
+      - "mq"
```

Comment:
Not sure why this was changed?

Reply:
Hmm, I also have no idea why this was changed. Resolved to be
```
      db:
        condition: "service_healthy"
      mq:
        condition: "service_healthy"
```